contents/docs/alerts/index.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Alerts
+sidebar: Docs
+showTitle: true
+availability:
+  free: partial
+  selfServe: full
+  enterprise: full
+---
+
+
+Alerts enable you to monitor your [insights](/product-analytics) and get notified when the value of the insight breaches a threshold that you've set (for example, number of unique visitors to your site increased by 5% week over week).
+Currently alerts are supported on all [trends](/docs/product-analytics/trends) (support for [funnels](/docs/product-analytics/funnels) coming soon!).
+
+
+## How to create an alert?
+1. Alerts are based off of [insights](/product-analytics). You can pick any existing trend insight or create a new one.
+2. Click on the "Alerts" button on top right of the insight page. This shows you all the alerts you've setup for this insight.
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/alerts_button_4ecd022cd6.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/alerts_button_4ecd022cd6.png"
+    alt="alerts button" 
+    classes="rounded"
+/>
+
+3. Click on 'New alert'
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/new_alert_button_30d3dde83c.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/new_alert_button_30d3dde83c.png"
+    alt="new alert button" 
+    classes="rounded"
+/>
+
+4. Set a name for the alert
+
+5. Pick which 'series' you want to alert on. These are the different events/values you're plotting in the insight. 
+    For example, an insight might have multiple series like `A - Pageviews` (total count of page views for a URL) and `B - Signups` (count of users who signed up on the page). 
+    You can pick either `A` or `B` which will determine if the alert gets checked against the value for `A` (total page views) or `B` (signups).
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/series_picker_aa01668862.png" 
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/series_picker_aa01668862.png"
+    alt="series picker" 
+    classes="rounded"
+/>
+
+
+6. Select the alert type. You can choose to check whether the series value against a specific value ('has value') or whether the value for the series has changed by a certain amount ('increases by'/'decreases by').
```

Comment:
I think there's something a little off with this phrasing:

> choose to check whether the series value against a specific value

It might be good to try bullet points for the options?

---

You can choose:

- "has value": checks the series against a specific value
- "increases by"/"decreases by": checks if the series has changed by a certain amount